test/crdsvalidation/konnect.konghq.com/konnectcloudgatewaytransitgateway_test.go

```diff
@@ -0,0 +1,234 @@
+package crdsvalidation
+
+import (
+	"testing"
+
+	"github.com/samber/lo"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	commonv1alpha1 "github.com/kong/kubernetes-configuration/api/common/v1alpha1"
+	konnectv1alpha1 "github.com/kong/kubernetes-configuration/api/konnect/v1alpha1"
+	"github.com/kong/kubernetes-configuration/test/crdsvalidation/common"
+)
+
+var konnectTransitGatewayTypeMeta = metav1.TypeMeta{
+	APIVersion: konnectv1alpha1.GroupVersion.String(),
+	Kind:       "KonnectCloudGatewayTransitGateway",
+}
```

Comment:
Can we move that also inside the test to not pollute the broader scope with this variable that's only used here?

Reply:
done.